Builds/VisualStudio/stellar-core.vcxproj.filters

```diff
@@ -1143,11 +1143,32 @@
     <ClCompile Include="..\..\src\util\Backtrace.cpp">
       <Filter>util</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\lib\spdlog\src\stdout_sinks.cpp">
```

Comment:
I think that those files have to be marked as "no compile" somehow, or we just don't include them in the project file.
I think only the `..\..\lib\spdlog.cpp` is needed.

Reply:
ok. I'll remove them from the project file.